packages/openapi-generator/src/parser/media-type.ts

```diff
@@ -23,10 +23,8 @@ export function parseApplicationJsonMediaType(
   options: ParserOptions
 ): OpenApiSchema | undefined {
   if (bodyOrResponseObject) {
-    const mediaType = getMediaTypeObject(
-      bodyOrResponseObject,
-      'application/json'
-    );
+    const allowedTypes = ['application/json', 'application/merge-patch+json'];
```

Comment:
[suggestion] Consider creating this variable as a constant outside the individual functions as it is re-used in two places.